armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -917,6 +917,19 @@ cat << 'EOF' > /etc/avahi/services/bitboxbase.service
 </service-group>
 EOF
 
+# firewall: restore iptables rules on startup
+cat << 'EOF' > /etc/systemd/system/iptables-restore.service
+[Unit]
+Description = BitBox Base: restore iptables rules
```

Comment:
Nit: Maybe we could be consistent with the spaces for systemd values in unit definitions? If we stay with the same style as elsewhere, this would be:
```
Description=BitBox Base: restore iptables rules
```